programs/util/strerror.c

```diff
@@ -55,11 +55,20 @@ int main( int argc, char *argv[] )
 {
     long int val;
     char *end = argv[1];
+#if defined(MBEDTLS_PLATFORM_C)
+    mbedtls_platform_context platform_ctx;
+    if( ( val = mbedtls_platform_setup( &platform_ctx ) ) != 0 )
+    {
+        mbedtls_printf( "  Failed initializing platform.\n" );
+        goto exit;
+    }
+#endif
```

Comment:
This is a utility program, that translates errors to string. No need for platform context initialization here.